app/popup/popup.js

```diff
@@ -196,6 +194,112 @@ function reloadPopup(tabID) {
       request.showLazyloadedImages();
     });
 
+    document.getElementById("parent-list").addEventListener("click", function (e) {
+      if (e.target && e.target.nodeName == "INPUT") {
+        jeclique(e.target.id);
+      }
+    });
+    
+    function jeclique(cookieName) {
```

Comment:
Could you rename this method with a more explicit name. I suggest `function toggleFlag(featureFlagName) {`